### PR TITLE
Bump version to 1.2.1 and build from source.

### PR DESCRIPTION
**Pull request by trzeciak**
See this issue: https://github.com/michaeleisel/zld/issues/72

### CODE REVIEW

File: Formula/zld.rb [hunk 1]
```diff
@@ -1,11 +1,14 @@
 class Zld < Formula
-  version "1.1.0"
+  version "1.2.1"
   desc "A faster version of ld, Apple's linker"
-  url "https://github.com/michaeleisel/zld/releases/download/#{version}/zld.zip"
-  sha256 "c606ec42d00b141576d2d2616ad3449a0ddca3edafd7917abdff063e919c8184"
+  url "https://github.com/michaeleisel/zld/archive/#{version}.zip"
+  sha256 "463faa979483a173d66c6204ce1289ddc05319ad70dfb0e685c1fbf9ca9ecd11"
```
Comment (michaeleisel) [inline Formula/zld.rb:5]: so the sha256 is now of the archive zip?

Comment (trzeciak) [reply Formula/zld.rb:5]: Yes, calculated like this:

```sh
wget https://github.com/michaeleisel/zld/archive/1.2.1.zip
sha256sum 1.2.1.zip
# 463faa979483a173d66c6204ce1289ddc05319ad70dfb0e685c1fbf9ca9ecd11  1.2.1.zip
```

**Comment by michaeleisel** [comment]
thinking more about this, my one concern is of updating it every release. right now, the "releases" page of zld is the source of truth on what the latest release is, but now we have to sync this one up as well. maybe people really like using brew to install though? i'm not sure

**Comment by trzeciak** [comment]
yea, if it will be required on every releases them yes it's poor xD
if it doesn't bother you, I can update it a few more times,
maybe in future we automate it and weave it to release process?
and yes, eg me I like use brew to install other development software

**Comment by trzeciak** [comment]
but in the other hand, this tool (zld) should be synchronised with Xcode version and a brew allows to install only a latest version of software(?)

**Pull request merged**